.github/workflows/java-pr-jobs.yml

```diff
@@ -9,13 +9,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    name: Test
+    name: Build and Test
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Test
-        run: mvn test
+      - name: Build and test
```

Comment:
If we want to test the correct command is `mvn verify`

Reply:
Shoot sorry - good spot, will update